spec/models/items/interlibrary_loan/interlibrary_loan_requests_spec.rb

```diff
@@ -20,42 +20,8 @@
         subject.each do |item|
           items = items + item.class.name
         end
-        expect(items).to eq('InterlibraryLoanRequest')
+        expect(items).to eq('')
       end
     end
   end
 end
```

Comment:
I'd make sure these tests still work for one loan or one request. I'd load illiad_request.json into a variable in before(:each), explicitly modify the transaction type to match. Then load it in the stub.